static/scripts/wp-stateless-setup.js

```diff
@@ -231,25 +231,6 @@ jQuery(document).ready(function($){
   var message = gs.find('#message');
   var projects_dropdown = gs.find('.projects');
   var bucketsWrapper = gs.find('#buckets-wrapper');
```

Comment:
@alimuzzaman We should be better at name spacing.. Use `wp-stateless-buckets-wrapper`, etc.